resources/assets/components/ReportbackUploader/index.js

```diff
@@ -72,9 +91,17 @@ class ReportbackUploader extends React.Component {
 
             <div className="wrapper">
               <div className="form-item">
-                <label className="field-label" htmlFor="caption">Add a caption for the photo.</label>
+                <label className="field-label" htmlFor="caption">Add a caption for the photo:</label>
                 <input className="text-field" id="caption" name="caption" type="text" onChange={this.handleOnCaptionUpdate} placeholder="60 characters or less" ref={input => (this.caption = input)} />
               </div>
+              <div className="form-item">
+                <label className="field-label" htmlFor="quantity">Update quantity:</label>
+                <input className="text-field" id="quantity" name="quantity" type="text" onChange={this.handleOnQuantityUpdate} placeholder="Optional if it already exists" ref={input => (this.quantity = input)} />
```

Comment:
do we need any validation around this to make sure this is an integer? If any text is submitted here, it changes the quantity to `0`

Reply:
@chloealee looks like that's the same thing that happens using the history modal too, so I could see validating both of those as a separate task

Reply:
@katiecrane that makes sense - great call! I created a new card for that here: https://www.pivotaltracker.com/n/projects/2019429/stories/152253258

Reply:
@chloealee one more question for you! I'm a little unfamiliar with how `onChange` works here. It will only call that function if they enter something into that field and then submit the form?

Reply:
yes if the value there changes @katiecrane !